static/css/shared/_ranking_unit.scss

```diff
@@ -18,11 +18,18 @@
  * Styling for ranking unit component.
  */
 
+.ranking-tile {
+  .chart-footnote {
+    margin: 8px 0 0 0;
+    font-weight: 500;
```

Comment:
Nit: Could you alphabetize the properties here and in the other scss blocks you edited?

Reply:
done

Reply:
Fixed!